src/applications/ivc-champva/10-7959C/config/form.js

```diff
@@ -213,53 +211,36 @@ const formConfig = {
       pages: {
         applicantNameDob: {
           path: 'applicant-information',
-          title: 'Applicant name and date of birth',
-          arrayPath: 'applicants',
+          title: formData =>
+            `${
+              formData.certifierRole === 'applicant' ? 'Your' : 'Applicant'
+            } name and date of birth`,
           uiSchema: applicantNameDobSchema.uiSchema,
           schema: applicantNameDobSchema.schema,
         },
-        applicantStart: {
-          path: 'applicant-information/:index/start',
-          arrayPath: 'applicants',
-          title: item => `${applicantWording(item)} information`,
-          showPagePerItem: true,
-          depends: () => !window.location.href.includes('review-and-submit'),
-          uiSchema: applicantStartSchema.uiSchema,
-          schema: applicantStartSchema.schema,
-        },
         applicantIdentity: {
-          path: 'applicant-information/:index/ssn',
-          arrayPath: 'applicants',
-          title: item => `${applicantWording(item)} identification information`,
-          showPagePerItem: true,
+          path: 'applicant-information/ssn',
+          title: formData =>
+            `${nameWording(formData)} identification information`,
           uiSchema: applicantSsnSchema.uiSchema,
           schema: applicantSsnSchema.schema,
         },
-        applicantAddressScreener: {
-          path: 'applicant-information/:index/pre-address',
-          arrayPath: 'applicants',
-          showPagePerItem: true,
-          keepInPageOnReview: false,
-          depends: (formData, index) => index > 0,
-          title: item => `${applicantWording(item)} mailing address`,
-          CustomPage: ApplicantAddressCopyPage,
-          CustomPageReview: null,
-          uiSchema: applicantPreAddressSchema.uiSchema,
-          schema: applicantPreAddressSchema.schema,
-        },
```

Comment:
This page allowed users to choose another applicant on the form with whom they shared an address. No longer needed.